godtossing/Pos/Ssc/GodTossing/Configuration.hs

```diff
@@ -50,3 +53,11 @@ mpcSendInterval = sec . fromIntegral . ccMpcSendInterval $ gtConfiguration
 mdNoCommitmentsEpochThreshold :: (HasGtConfiguration, Integral i) => i
 mdNoCommitmentsEpochThreshold =
     fromIntegral . ccMdNoCommitmentsEpochThreshold $ gtConfiguration
+
+-- | In the first mainnet version we messed up the calculation of the
+-- initial leaders set, and the leaders weren't sending commitments. It has
```

Comment:
Why _leaders_? Richmen ought to send commitments, not leaders.

Reply:
Oops, brainfart.